### PR TITLE
Style book-store app

### PR DESCRIPTION
**Pull request by lawrahkonwea**
In this milestone I achieved the following;

- Added missing JSX elements
- Styled the website to using Zeplin's design

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "prop-types": "^15.8.1",
```
Comment (MeqdamAlqudah) [inline package.json:12]: - [ ] Please add the title of the project in the Readme file 
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @lawrahkonwea , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.
image


 ## Highlights
:+1: no linter errors 
:+1: styling is good it looks like the design 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -23,10 +23,10 @@
```
Comment (HAMMAS-SALEEM) [inline README.md:23]: image

**[Optional]**

- [ ] This section should have a message like **If you like this project then don't forget to give a star :star: to this repository** to make your readme descriptive :+1:

Comment (HAMMAS-SALEEM) [inline README.md:23]: **[Optional]**
- [ ] You don't have an **Acknowledgment** section in your readme file :thinking: Kindly put an **Acknowledgement** section to make your readme file professional :+1:

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

### Hey @lawrahkonwea 👋

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Requirements
:heavy_check_mark: Following Gitflow

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**